_posts/en/newsletters/2020-01-15-newsletter.md

```diff
@@ -0,0 +1,140 @@
+---
+title: 'Bitcoin Optech Newsletter #80'
+permalink: /en/newsletters/2020/01/15/
+name: 2020-01-15-newsletter
+slug: 2020-01-15-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter requests help testing the next version of LND,
+summarizes a discussion about soft fork activation mechanisms, and
+describes a few notable changes to popular Bitcoin infrastructure
+software.
+
+## Action items
+
+- **Help test LND 0.9.0-beta-rc1:** this [pre-release][lnd 0.9.0-beta]
+  of the next major version of LND brings several new features and bug
+  fixes.  Experienced users are encouraged to help test the software so
+  that any problems can be identified and fixed prior to release.
+
+## News
+
+- **Discussion of soft fork activation mechanisms:** Matt Corallo
+  started a [discussion][corallo sf] on the Bitcoin-Dev mailing list about
+  what attributes are desirable in a soft fork activation method and
+  submitted a proposal for a mechanism that contains those attributes.
```

Comment:
FWIW I followed this ML exchange with interest, but wondered what the context was for launching the discussion. Was there an event, a trigger, something in particular?

Reply:
There was a bit of discussion about what upgrade mechanism to use for taproot a few weeks ago; this didn't involve Matt (although he was mentioned): http://www.erisian.com.au/meetbot/taproot-bip-review/2019/taproot-bip-review.2019-12-17-19.01.log.html#l-56

But I don't know for sure whether that had anything to do with Matt's post, so I hesitate to link to it.

Reply:
Thanks @harding. I agree that it's too tenuous to link to it.